pkg/Library/Graph/Atoms/Layer.js

```diff
@@ -0,0 +1,28 @@
+export const atom = (log, resolve) => ({
+/**
+ * @license
+ * Copyright 2023 NeonFlan LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+async update({graph}, state, {service}) {
```

Comment:
i think this is brilliant - Layer is just another Node with an Id and a slot... wow (i didn't fully grasp it yesterday from the screenshot, and it is soooo elegant)!

Reply:
🚀